### DISCUSSION TITLE
Conditional line_break based on TERM_PROGRAM (on MacOS)

### DISCUSSION THREAD

**Discussion by dskyberg** [starship/starship#6022] [Q&A]
I use [Warp](https://www.warp.dev/) as my terminal. Warp adds an edit box below the prompt.  So, Starship `line_break=false` is appropriate.  However, for other terminals (such as within VSCode) I would like `line_break=true`.  Is there any way to configure this in starship.toml?

**Comment by agzam** [comment]
This annoyed me, broke vterm for me in Emacs, making it impossible to type. I wish I could do it conditionally, based on the terminal type.
